.github/workflows/swift-toolchain.yml

```diff
@@ -560,6 +559,11 @@ jobs:
     name: ${{ matrix.os }} ${{ matrix.arch }} CMark GFM
 
     steps:
+      - uses: actions/checkout@v4.2.2
+        with:
+          path: ${{ github.workspace }}/SourceCache/swift-build
+          show-progress: false
```

Comment:
We have to check this out for the action I take it?

Reply:
Yes, that's correct. It's probably the main drawback with this approach